autodistill/detection/classification_base_model.py

```diff
@@ -0,0 +1,65 @@
+from abc import abstractmethod
+from dataclasses import dataclass
+
+import os
+import glob
+from tqdm import tqdm
+import cv2
+
+import supervision as sv
+from autodistill.core import BaseModel
+from autodistill.detection import ClassificationOntology
+
+from autodistill.helpers import split_data
+
+@dataclass
+class ClassificationBaseModel(BaseModel):
+    ontology: ClassificationOntology
+
+    @abstractmethod
+    def predict(self, input: str) -> sv.Detections:
```

Comment:
Does this return a Detection? Or something Classification specific?

Reply:
I have changed this to return an `sv.Classifications` object (https://roboflow.github.io/supervision/classification/core/#classifications).